doc/api/globals.md

```diff
@@ -583,6 +583,41 @@ The `MessagePort` class. See [`MessagePort`][] for more details.
 
 This variable may appear to be global but is not. See [`module`][].
 
+## `Navigator`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+An implementation of the [Navigator API][].
+
+## `navigator`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+An implementation of [`window.navigator`][].
```

Comment:
On a closer look, I think the [`WorkerNavigator`](https://html.spec.whatwg.org/multipage/workers.html#the-workernavigator-object) interface might make more sense for us compared to [`Navigator` ](https://html.spec.whatwg.org/multipage/system-state.html#the-navigator-object). Or at least that seems to be a better starting point compared to `Navigator`. (we already run WPT in workers, too).

Reply:
Why not support both? I suggest adding WorkerNavigator in a different pull request.

Reply:
I would find it confusing to expose both of them: every Web interface has an `Exposed` attribute defining the scope where they are available. `WorkerNavigator` is only available with `WorkerGlobalScope` and `Navigator` is only available with `WindowGlobalScope`. Not both.

Reply:
You're right. How can I limit `window.navigator` to window, not worker?

Reply:
IIRC, we didn't distinguish `WindowGlobalScope` and `WorkerGlobalScope` yet as no [`WindowGlobalScope`](https://html.spec.whatwg.org/multipage/nav-history-apis.html#window) specific APIs are exposed.

Replying to the question "Why not support both?". I'd agree that `WorkerNavigator` is more limited than `Navigator` and suits the use case of the PR, since `WorkerNavigator` doesn't include [`NavigatorContentUtils`](https://html.spec.whatwg.org/multipage/system-state.html#navigatorcontentutils), [`NavigatorCookies`](https://html.spec.whatwg.org/multipage/system-state.html#navigatorcookies), and [`NavigatorPlugins`](https://html.spec.whatwg.org/multipage/system-state.html#navigatorplugins).